packages/client-sdk-nodejs/src/cache-client.ts

```diff
@@ -1191,11 +1185,11 @@ export class CacheClient extends AbstractCacheClient implements ICacheClient {
     );
   }
 
-  private getNextDataClient(): DataClient {
+  protected getNextDataClient(): DataClient {
     const client = this.dataClients[this.nextDataClientIndex];
     this.nextDataClientIndex =
       (this.nextDataClientIndex + 1) % this.dataClients.length;
-    return client;
+    return client as DataClient;
```

Comment:
once the pair of nodejs/web sdks are at parity, this type cast will no longer be needed